### PR TITLE
Added configuration page

### PR DESCRIPTION
**Pull request by enrrou**
Resolves https://github.com/opentelekomcloud-infra/docsportal/issues/3

### CODE REVIEW

**Comment by enrrou** [comment]
I think, we should reformat the doc  moving tool into separate doc pages.
This will make easier to navigate the docportal.

**Review by gtema — CHANGES_REQUESTED** [review]
Well, direction is right. But as I wrote, I want configuration to be present here cause it is valid for all of our tools (ansible, tf, cli, python, etc), and not only for the CLI (please remove focus of the CLI). And as described in https://github.com/opentelekomcloud-infra/docsportal/issues/3 it should be also extended with content from openstacksdk docs that we are currently missing (caching)

File: doc/source/configuration.rst [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+      auth:
+        username: '<USER_NAME>'
+        project_name: '<eu-de_project>'
+        # or project_id: '<123456_PROJECT_ID>'
+        user_domain_name: 'OTC00000000001000000xxx'
+        # or user_domain_id: '<123456_DOMAIN_ID>'
+        auth_url: 'https://iam.eu-de.otc.t-systems.com:443/v3'
+      interface: 'public'
+      identity_api_version: 3 # !Important
+      ak: '<AK_VALUE>' # AK/SK pair for access to OBS
```
Comment (gtema) [inline doc/source/configuration.rst:128]: would you please also place sk into the secrets.yaml section here? We want to show proper case from security pov

File: doc/source/configuration.rst [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+        # or target_domain_name: '<123456_DOMAIN_NAME'
+        target_agency_name: 'test_agency' # name of the agency
+        target_project_name: '<123456_PROJECT_NAME>' # project scoped operations
+        # or target_project_id: '<123456_PROJECT_ID>'
+        # When target_project_xx is not set - domain scope is selected
+
+Configuration of Environment Variables
+--------------------------------------
+
+Instead of using the clouds.yaml file, environmnt variables can be configured
```
Comment (gtema) [inline doc/source/configuration.rst:172]: Please rephrase sentence here that it somehow does not really advertise to use this way as a preferred one. Maybe something in the form: `historically OpenStack tools are supporting configuration through environment variables...`.

File: doc/source/configuration.rst [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+        project_name: <eu-de_PROJECT_NAME>
+      region_name: eu-de
+      dns_api_version: 1
+
+`openstacksdk` can also cache authorization state (token) in the keyring.
+That allow the consequent connections to the same cloud to skip fetching new
+token. When the token gets expired or gets invalid `openstacksdk` will
+establish new connection.
+
+
```
Comment (gtema) [inline doc/source/configuration.rst:266]: can you please also include here the MFA configuration case (https://docs.openstack.org/openstacksdk/latest/user/config/configuration.html#mfa-support)?

File: doc/source/index.rst [hunk 4]
```diff
@@ -23,16 +24,19 @@ SDK
...
-
 Other Drivers and Tools
 -----------------------
 
 * `Docker machine driver <https://github.com/opentelekomcloud/docker-machine-opentelekomcloud>`_
 * `Rancher Open Telekom Cloud CCE driver <https://github.com/opentelekomcloud/kontainer-engine-driver-otc>`_
+
+Cloud Connect Configuration
+---------------------------
+OTC Connect is easy to :doc:`configure <configuration>`. All credentials can be
```
Comment (gtema) [inline doc/source/index.rst:41]: I dislike term `OTC Connect`. We can instead write `Open Telekom Cloud tools are easy to configure ...`

**Review by gtema — CHANGES_REQUESTED** [review]
Sounds already much better. But please still address further comments

File: doc/source/configuration.rst (hunk 1 above)
Comment (enrrou) [reply doc/source/configuration.rst]: done

File: doc/source/configuration.rst (hunk 2 above)
Comment (enrrou) [reply doc/source/configuration.rst]: rephrased

File: doc/source/configuration.rst (hunk 3 above)
Comment (enrrou) [reply doc/source/configuration.rst:266]: added mfa

File: doc/source/index.rst (hunk 4 above)
Comment (enrrou) [reply doc/source/index.rst]: done

File: doc/source/index.rst [hunk 5]
```diff
@@ -23,16 +24,19 @@ SDK
...
 Other Drivers and Tools
 -----------------------
 
 * `Docker machine driver <https://github.com/opentelekomcloud/docker-machine-opentelekomcloud>`_
 * `Rancher Open Telekom Cloud CCE driver <https://github.com/opentelekomcloud/kontainer-engine-driver-otc>`_
+
+Cloud Connect Configuration
+---------------------------
+Open Telekom Cloud tools are easy to :doc:`configure <configuration>`. All credentials can be
+configured conveniently in a few files. OTC connect configuration is general for all tools above.
```
Comment (gtema) [inline doc/source/index.rst:42]: still one occurrence of `OTC connect` remaining.

Comment (enrrou) [reply doc/source/index.rst]: thanks, fixed

**Review by gtema — DISMISSED** [review]

File: doc/source/configuration.rst [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+    $ openstack flavor list
+
+.. note::
+   You don't need to specify the `--os-cloud` parameter when environment
+   variables are used.
+
+Cache Settings
+--------------
+
+Accessing a cloud is often expensive, so it's quite common to want to do some
```
Comment (tischrei) [inline doc/source/configuration.rst:221]: I suggest the following phrasing: '[...]quite common to use client-side caching[...]'

**Review by tischrei — APPROVED** [review]
There is only one, not so important phrasing suggestion which leads in an approval from my side.

File: doc/source/configuration.rst (hunk 6 above)
Comment (enrrou) [reply doc/source/configuration.rst]: thanks, it's a nice advice.

**Pull request merged**